### DISCUSSION TITLE
Systemvoraussetzungen für das neueste Release 21.12.14

### DISCUSSION THREAD

**Discussion by Claudiawa** [PapooSoftware/PapooCMS#9] [Q&A] [answered]
Hallo zusammen,
welche Systemvoraussetzungen benötigt das Release 21.12.14?
Auf www.papoo.de steht bei den technischen Anforderungen ab PHP-Version 7.0. Welche PHP-Version benötigt Papoo minimal und welche höchste Version wird unterstützt?
Vielen Dank und viele Grüße
Claudia

**Comment by ah-papoo** [comment, marked as answer]
Hallo Claudia,
Von PHP 7.x bis 8.0 ist das getestet und funktioniert. Mit PHP 8.1 ist das CMS noch nicht getestet worden, damit funktionieren könnte es aber.
Grüße Achim
